src/Traits/HasRoles.php

```diff
@@ -28,6 +28,43 @@ public function scopeWithRoles($query)
     {
         return $query->with('roles');
     }
+    
+    /**
+     * Determine if any of the assigned roles to this user
+     * have a specific role.
+     *
+     * @param string $role
+     *
+     * @return bool
+     */
+    public function hasRoleTo($role)
```

Comment:
The method name seems confusing. I think it should be `hasRole` instead. While passing the role name as a string may work for some situations, it isn't exactly flexible. How about something like this: https://github.com/spatie/laravel-permission/blob/2512d5a22aaae0bab8d62a02e502a603776da9da/src/Traits/HasRoles.php#L199

Reply:
Yeah i agree i just follow the pattern of you permission naming but will change